ra/ra.go

```diff
@@ -972,6 +972,10 @@ func (ra *RegistrationAuthorityImpl) failOrder(
 // If successful the order will be returned in processing status for the client
 // to poll while awaiting finalization to occur.
 func (ra *RegistrationAuthorityImpl) FinalizeOrder(ctx context.Context, req *rapb.FinalizeOrderRequest) (*corepb.Order, error) {
+	if req == nil || req.Order == nil {
+		return nil, berrors.InternalServerError("Incomplete gRPC request message")
```

Comment:
Sorry, shoulda realized this in the last round of comments. This file already has this message as `errIncompleteGRPCRequest` on line 52; lots of other already-unwrapped functions are using it already.

Reply:
No worries! Thanks for catching this 👍🏻